test/specs/validate-address.spec.js

```diff
@@ -804,7 +804,7 @@ describe("validateAddress()", () => {
         source: "shipengine",
         type: "system",
         code: "unspecified",
-        message: "Unable to connect to the database",
+        message: "Unable to process this request. A downstream API error occurred.",
```

Comment:
In the new simengine, we mask all downstream errors to this string as to not leak internal information to the outside world.

Reply:
Cool.  Just be sure that you set the `type` field appropriately.  So if the error occurred in a downstream carrier API then `type` should be `carrier`, etc.